airflow/providers/amazon/aws/example_dags/example_s3_to_sql.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Following up on the discussion here #22438, this file should be moved in `tests/system/providers/amazon/aws/example_s3.py`

Reply:
@vincbeck I have modified the example DAG according to AIP-47, but I was not able to test it yet. Will do it as soon as I have fixed my breeze environment

Reply:
The system test looks really good! Thanks for the effort to have converted the example DAG!

Reply:
@vincbeck I added a SQL Check operator to count the rows inserted. Tests were running fine locally.
Can you give me a hint which conn_id's I have to use?
For S3 topics I guess `aws_default`, but for generic sql? `sql_default`?